templates/core/terraform/resource_processor/vmss_porter/main.tf

```diff
@@ -163,3 +163,11 @@ resource "azurerm_role_assignment" "subscription_owner" {
   role_definition_name = "Owner"
   principal_id         = azurerm_user_assigned_identity.vmss_msi.principal_id
 }
+
+resource "azurerm_key_vault_access_policy" "resource_processor" {
```

Comment:
What is this used for?

Reply:
Gitea deployment doesn't work otherwise -- the deploying entity needs access to a secret (I can't remember which one it was but probably the administrator password)